bats/tests/profile/create-profile-output.bats

```diff
@@ -0,0 +1,424 @@
+load '../helpers/load'
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start app' {
+    start_container_engine
+    wait_for_container_engine
+    rdctl set --images.namespace=change-to-k8s.io
```

Comment:
I wonder if we should use a global variable for the namespace, so we don't have to repeat the constant later in the tests. Can just be declared at the file scope (or in a `local_setup`, if you prefer):

```
CUSTOM_NAMESPACE=change-to-k8s.io
```

Reply:
ok